### ISSUE TITLE
Bug on home loading[Todoist] ...

### ISSUE THREAD

**Issue by jpalmerov** [raycast/extensions#14364]
### Extension

https://www.raycast.com/doist/todoist

### Raycast Version

1.82.3

### macOS Version

14.5

#### Error:
```
TypeError: Cannot read properties of null (reading 'due')

Um:TodayTasks.tsx:16:51

---
13: 
14: type TodayTasksProps = { quickLinkView: QuickLinkView }

log:
XD a NullPointerException

TypeError: Cannot read properties of null (reading 'due')

Um:TodayTasks.tsx:16:51

---
13: 
14: type TodayTasksProps = { quickLinkView: QuickLinkView };
15: 
16: export default function TodayTasks({ quickLinkView }: TodayTasksProps) {
17:   const [data] = useCachedData();
18: 
19:   const tasks = useMemo(() => {
---

Lr:index.js:8:3631
    at ray-list
lt:index.js:20:24444

### Steps To Reproduce

1. open home

### Current Behaviour

nothing there

### Expected Behaviour

something there

**Comment by neehhaa06** [comment]
Hello!! @jpalmerov :)

fix: #14364

The `TypeError` occurs because `data` is `null`, leading to attempts to read properties like `due` on it. To fix this, ensure `data` is not `null` before accessing its properties. Add a conditional check in the `TodayTasks` component to handle the null case and prevent accessing properties on null objects. For example, include a check like `if (!data) return <div>No data available</div>;` before proceeding with property access.

**Comment by thomaslombart** [comment]
Thanks for reporting this bug. Would you like to work on it? If yes, here's our [guide on extension contributions](https://developers.raycast.com/basics/contribute-to-an-extension) so that you can open up a pull request. Feel free to contact me here or in the [Slack community](https://raycast.com/community) if you need further help.
